.ci/Jenkinsfile

```diff
@@ -295,7 +295,9 @@ pipeline {
 
 def unpack_artifacts(builds) {
     builds.values().each() { build ->
-      copyArtifacts(projectName: "apm-agent-ruby/apm-agent-ruby-downstream/${env.BRANCH_NAME}", selector: specific(buildNumber: build.number.toString()))
+      copyArtifacts(projectName: "apm-agent-ruby/apm-agent-ruby-downstream/${env.BRANCH_NAME}", 
+                    filter: 'coverage/**/*.*',
+                    selector: specific(buildNumber: build.number.toString()))
```

Comment:
Let's only care about the coverage ones as that's the aim of this particular section. This will help to avoid duplicating the test results when aggregating the dowsntream test output that happens later on.